system-tests/README.md

```diff
@@ -1,122 +1,260 @@
-## System tests
+# System tests
+
+The system tests copies a file from a provider to a consumer blob storage account.
+
+## Publish/Build Tasks
+
+> ! Important Note !
+> 
+> MVD depends on Eclipse DataSpaceConnector(EDC), Identity Hub and Registration Service. These dependencies 
+> are __not__ published to any central artifact repository yet, so in local development we have to use locally 
+> published dependencies.
+>
+>In order to use the correct version of each repo required by the `MVD`, you need to look in [action.yml](./.github/actions/../../../.github/actions/gradle-setup/action.yml) for the hashes of the versions of the `EDC`, `Identity Hub` and the `Registration Service` that are being used by the `MVD`.
+>
+> For Example, the `Registration Service`, `Identity Hub` and `EDC` repository hash can be found in the _Checkout_ steps  (in the `ref` property) of [action.yml](./.github/actions/../../../.github/actions/gradle-setup/action.yml):
+
+```yml
+    - name: Checkout EDC
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/DataSpaceConnector
+        path: DataSpaceConnector
+        ref: 3ff940b720f44826df28e893fb31344eb6faacef
```

Comment:
should we remove these values so that people don't believe they should get them from here

Reply:
The instructions indicate it is an example and show where to get them from; the `ref` value could be removed, or updated as needed until the time that it is not longer needed (when dependencies are published)

Reply:
Yes I got that, I just expect that we will forget to update them / users will wrongly take them from here, hence I think it's better to remove these values from here